### PR TITLE
Add Rocky Linux

### PR DESCRIPTION
**Pull request by NeilHanlon**
Hello,

We'd like to add rockylinux shortnames for our library image.

Please let us know if there's anything we can do to help, and thank you in advance!

Best,
Neil

### CODE REVIEW

**Review by saschagrunert — APPROVED** [review]
LGTM

File: shortnames.conf [hunk 1]
```diff
@@ -54,6 +54,9 @@
   "ubi8/ubi-minimal" = "registry.access.redhat.com/ubi8-minimal"
   "ubi8/ubi-init" = "registry.access.redhat.com/ubi8-init"
   "ubi8/ubi-micro" = "registry.access.redhat.com/ubi8-micro"
+  # Rocky Linux
+  "rockylinux" = "docker.io/library/rockylinux"
+  "rocky" = "docker.io/library/rockylinux"
```
Comment (vrothberg) [inline shortnames.conf:59]: I am not sure about `rocky` since it's a different name but let @fatherlinux and @rhatdan weigh in.

**Review by vrothberg — COMMENTED** [review]
Thanks! Just one nit.

**Comment by fatherlinux** [comment]
Yeah, I'd probably keep them the same. Or add two entries.

File: shortnames.conf (hunk 1 above)
Comment (NeilHanlon) [reply shortnames.conf:59]: > Yeah, I'd probably keep them the same. Or add two entries.

Sure, i'll remove the `rocky` tag for now. 

Thanks!

**Comment by rhatdan** [comment]
LGTM

**Pull request merged**